railties/lib/rails/engine/commands.rb

```diff
@@ -1,12 +1,7 @@
-require "rails/command"
+unless defined?(APP_PATH)
+  if File.exists?(File.expand_path("test/dummy/config/application.rb", ENGINE_ROOT))
+    Object.const_set(:APP_PATH, File.expand_path("test/dummy/config/application", ENGINE_ROOT))
```

Comment:
Sorry, this is a remnant of when I was fixing variously, `const_set` is unnecessary in the current code. I removed `const_set`.